pkg/imexport/export.go

```diff
@@ -73,11 +162,15 @@ func export(tw *tar.Writer, fs vfs.VFS) error {
 
 		return nil
 	})
+	if err != nil {
+		return err
+	}
+	err = metadata(tw, fs, domain)
```

Comment:
You can shorten the last 2 lines of this function in `return metadata(tw, fs, domain)`